### DISCUSSION TITLE
KICS version 1.5.7 include-queries not working

### DISCUSSION THREAD

**Discussion by darylmcnutt** [Checkmarx/kics#5328] [Q&A] [answered]
Greetings! 

I've been working with KICS for the last few months and suddenly the `include-queries` flag is not working. I've tested using it in the configuration file (which has been working for me for some time) and I've tried using it as a CLI flag - Both of which successfully runs the scan, but the out put is `Queries loaded: 0` when 50+ queries are listed. 

Is anybody else having a similar issue? I haven't made any major modifications to my Gitlab CI/CD pipeline or jobs. 

Note: The `include-queries` option seems to have stopped working for me after KICS version 1.5.7 was released. 

![image](https://user-images.githubusercontent.com/105223550/167437135-1e10e99d-5b10-4afe-8c54-e808308253e8.png)
![image](https://user-images.githubusercontent.com/105223550/167437264-0087d1ca-f8d3-4c2b-abfd-f5fbe4e6d52b.png)

**Comment by cx-nuno-oliveira** [comment, marked as answer]
Hi @darylmcnutt, from your screen shots the queries you include are terraform and cloudformation. Do the files you scan belong to any of these technologies? Did you have results on these queries?

In version 1.5.7 we changed -t (--type) flag to actually analyze the files being scanned to discard them when they match the extension but the content is not of the type selected (e.g., ordinary json file vs cloudformation json file). In case we don't find any suitable file, we won't load queries as well.

Hope this helps.
